src/components/views/LobbyOverview.tsx

```diff
@@ -54,60 +56,109 @@ const LobbyOverview = () => {
         setSelectedLobby((prevSelectedLobby) =>
             prevSelectedLobby === lobby ? null : lobby
         );
+        setLobbyCode(lobby.lobbyCode);
     };
-    /*
-    1. Take previous input
-    2. Verify if previous input === current input: If yes, set lobby to null
-    3. Else, set lobby to the current lobby*/
 
-    //Allows toggling, true/false
     const iconClick = () => {
         setLoginRegisterPopup((prevState) => !prevState);
     };
-    /*
-    1. Take previous state
-    2. Return the negation of it. (Allows toggling of LoginRegisterPopup variable)*/
 
-    let content = <Spinner />;
+    const joinLobby = async () => {
+
+        try {
+            const requestbody = [];
+
+            /* For some reason it is necessary to make the post request with
+             * a requestbody, if i leave it out, the post request returns an error. */
+
+            const config = {headers: {userToken: userToken}};
+            const response = await api.post("/lobbies/" + lobbyCode + "/players", requestbody, config);
+            console.log(response);
+            navigate("/lobby/" + lobbyCode)
+        } catch (error) {
+            handleError(error);
+            alert(("Unable to join lobby with lobbycode: " + lobbyCode));
```

Comment:
add some more specific error handling or/and display error message that was sent from the server so the user knows what the reason for the failure was and can take action (logging in for example)